### ISSUE TITLE
 yii\base\ErrorException: Cannot use 'Object' as class name as it is reserved in /var/www/html/admin.gt-music-app.com/vendor/yiisoft/yii2/base/Object.php:28

### ISSUE THREAD

**Issue by igorveremsky** [hiqdev/yii2-hiart#15]
after install extension get this error:
![screenshot_315](https://user-images.githubusercontent.com/13087365/41980972-83defaf4-7a30-11e8-86e0-6f8def3e42d9.png)
Yii debug log:
![screenshot_316](https://user-images.githubusercontent.com/13087365/41981043-aee96388-7a30-11e8-847d-b25669964255.png)
Api connection config:
```php
	    'hiart' => [
		    'class' => \hiqdev\hiart\rest\Connection::class,
		    'requestClass' => \hiqdev\hiart\auto\Request::class,
		    'baseUri' => 'http://api.gt-music-app.com/v1/',
	    ],
```
Genre Model:
```php
<?php

namespace app\models;

/**
 * Class Genre
 * @package app\models
 *
 * @property int $id
 * @property string $name
 */
class Genre extends \hiqdev\hiart\ActiveRecord
{
	/**
	 * {@inheritdoc}
	 */
	public function rules()
	{
		return [
			['id', 'integer', 'min' => 1],
			[['name'], 'required'],
			[['name'], 'string', 'max' => 255],
			[['name'], 'unique'],
		];
	}
}
```
Genre Controller Action:
```php
    /**
     * Lists all Genre models.
     * @return mixed
     */
    public function actionIndex()
    {
    	$genre = new Genre();
    	$genre->name = 'test';

    	$genre->save();
    }
```
Someone have idea how to solve it?

**Comment by igorveremsky** [comment]
okey, it was easy :) as described at class PHPDoc
```@deprecated since 2.0.13, the class name `Object` is invalid since PHP 7.2, use [[BaseObject]] instead.```
but ```AbstractQueryBuilder``` extends from it at ```0.2.0``` version so fatal errror throws.
To solve it require section at ```composer.json``` from ```"hiqdev/yii2-hiart": "~0.2.0"``` to ```"hiqdev/yii2-hiart": "dev-master"```. Think developers need to change info at README.MD that stable version is 0.2,0

**Comment by hiqsol** [comment]
I just released `0.3.0` version. Stable is the latest state. We use it in our production.
